.github/dependabot.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+updates:
+  # Dependencies listed in go.mod
+  - package-ecosystem: "gomod"
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+
+  # Dependencies listed in .github/workflows/*.yml
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      
```

Comment:
Can you also add a comment here to briefly explaining what the function does?

Reply:
```suggestion
      # Dependencies listed in Dockerfile
```